packages/EasyActivity/src/Bridge/Symfony/Serializers/CircularReferenceHandlerInterface.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyActivity\Bridge\Symfony\Serializers;
+
+interface CircularReferenceHandlerInterface
+{
+    /**
+     * @param mixed[] $context
+     *
+     * @return mixed
+     */
+    public function __invoke(object $object, string $format, array $context);
```

Comment:
We always return `string` here. What about changing the declaration to the strict return type?

Reply:
done